### DISCUSSION TITLE
Enhancement: make building on switch a bit more user friendly

### DISCUSSION THREAD

**Discussion by arceuss** [FunkinCrew/Funkin#1611] [Ideas]
you could use something like [https://github.com/retronx-team/switch-haxe](https://github.com/retronx-team/switch-haxe) but i havent tried it

**Comment by arceuss** [comment]
but i also found [https://www.ratalaikagames.com/spiller.php](https://www.ratalaikagames.com/spiller.php) but its paid i think

**Comment by suprstarrd** [comment]
It should *not* be hard to get a Nintendo Developer account (which I literally did just a few weeks ago!), request access to the Nintendo Switch tools, and get accepted. But if that's too hard, there's a reason that it's not user friendly-- it's meant for an NSP that you can upload on the eShop!

**Comment by GStudiosX2** [comment]
> It should *not* be hard to get a Nintendo Developer account (which I literally did just a few weeks ago!), request access to the Nintendo Switch tools, and get accepted. But if that's too hard, there's a reason that it's not user friendly-- it's meant for an NSP that you can upload on the eShop!

Lol what happens tho if they start to deny most requests just because random people just want access to the api lol

**Comment by GStudiosX2** [comment]
You should suggest this to Nintendo tho not here because Nintendo made the api not funkin
